api/storage_service.cc

```diff
@@ -294,6 +294,13 @@ void set_repair(http_context& ctx, routes& r, sharded<repair_service>& repair) {
                 });
     });
 
+    ctx.db.local().get_builtin_routine_registry().register_routine("keyspace_repair",
+            db::make_single_value_routine<int>("repair", [&repair](std::unordered_map<sstring, sstring>&& val) {
+        auto ks = val["ks"];
```

Comment:
Shouldn't `ks` be checked for existence and validated?
Also, is it stored under "ks", not "keyspace"?